configuration/configurator/SchemaSequence.cpp

```diff
@@ -0,0 +1,119 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#include <cassert>
+#include "jptree.hpp"
+#include "XMLTags.h"
+#include "SchemaSequence.hpp"
+#include "SchemaElement.hpp"
+#include "JSONMarkUp.hpp"
+
+using namespace CONFIGURATOR;
+
+#define StringBuffer ::StringBuffer
+#define IPropertyTree ::IPropertyTree
```

Comment:
If added to the header file any file that includes them would have that definition as well, which is not always correct.  This should really be fixed in the platform to properly namespace HPCC jlib libraries, they should not live in global namespace.